### PR TITLE
Sphinx documentation skeleton

### PR DESCRIPTION
**Pull request by delucchi-cmu**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
+
+master_doc = "index"  # This assumes that sphinx-build is called from the root directory
+html_show_sourcelink = (
+    False  # Remove 'view source code' from top of page (for html, not python)
+)
+# -- Options for HTML output -------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
+
+html_theme = "alabaster"
```
Comment (drewoldag) [inline docs/conf.py:26]: I (personally) would recommend not using 'alabaster' here. I just don't like it. Also, it's a bit strange to generate the docs locally using sphinx_rtd_theme, and have the actual documentation look different on readthedocs rendered with 'alabaster'. 

So I think that line 26 can be removed, and the conditional on line 34 and 35 can be as well.

File: docs/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+sphinx==6.1.3
+sphinx_rtd_theme==1.2.0
+sphinx-autoapi==2.0.1
```
Comment (drewoldag) [inline docs/requirements.txt:3]: I don't think we need the `sphinx-autoapi`, it was for automatically generating docs from docstrings. And I don't think that will be applicable for this documentation.

File: docs/source/contributing.rst [hunk 3]
```diff
@@ -0,0 +1,43 @@
+Contributing to the template
+===============================================================================
+
+Find (or make) a new GitHub issue
+-------------------------------------------------------------------------------
+
+Add yourself as the assignee on an existing issue so that we know who's working on what. ( If you're not actively working on an issue, unassign yourself 😉 )
```
Comment (drewoldag) [inline docs/source/contributing.rst:7]: I think we should include a statement like, "If you don't have permissions to assign yourself to an issue, please reach out to us on #lincc-dev (maybe?) or lincc-frameworks-team@lists.lsst.org".

**Review by drewoldag — APPROVED** [review]
This looks great. Thank you for putting this together @delucchi-cmu !

File: docs/source/contributing.rst (hunk 3 above)
Comment (rmandelb) [reply docs/source/contributing.rst:7]: If we refer to #lincc-dev we'd have to say "in the LSSTC Slack workspace", which not everybody who sees this page will have access to.  I'm not sure if @jeremykubica configured the team list to be reachable by people who aren't subscribed to it (is it meant for team discussion or for people outside the team to reach the entire team?).  

I think we may want to think carefully about how we encourage people not on the team to reach out - this applies, as well, for people with questions on the website or with questions about any of our GitHub repos etc.  Do we already have a consistent statement on this elsewhere that we can use here?

File: docs/conf.py (hunk 1 above)
Comment (delucchi-cmu) [reply docs/conf.py:26]: I copied this from the python project template, so maybe we should update it there too =]

File: docs/source/contributing.rst (hunk 3 above)
Comment (delucchi-cmu) [reply docs/source/contributing.rst:7]: For now, we can recommend folks create a new issue to start a conversation.

**Pull request merged**